lib/shimmer/tasks/db.rake

```diff
@@ -20,6 +25,15 @@ namespace :db do
 
   desc "Downloads the app assets from Heroku to directory `storage`."
   task pull_assets: :environment do
+    unless system("which aws > /dev/null 2>&1")
+      if system("which brew > /dev/null 2>&1")
+        puts "AWS CLI is not installed. You can install it using Homebrew with the command: brew install awscli"
+      else
+        puts "AWS CLI is not installed. Please install it to continue."
+      end
+      exit(1)
```

Comment:
the two additions are inconsistent. one exists, the other doesnt

Reply:
I added a matching heroku-cli check to the pull_data task. 

**pull_data** will not check for the heroku-cli, and give brew instructions to install the required package as well as check to make sure there is a linked remote repo, if not, it will remind the user to link the heroku remote.

**pull_assets** will do something similiar except check if aws is installed and if not it will recommend a brew install or just tell the user to install it on their system.

Does this address your concerns?